<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAddProDlg</class>
 <widget class="QDialog" name="CAddProDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目添加/编辑</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>范围类型：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>工人填入：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDIT_PRONAME">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>15</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="RDO_ZYS">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>62</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>总印数</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RDO_ZLS">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>62</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>总令数</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RDO_YES">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>102</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>是</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RDO_NO">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>102</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SAVE">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
